components/sponsored_benefits/app/models/sponsored_benefits/services/plan_cost_service.rb

```diff
@@ -75,7 +75,7 @@ def monthly_employee_costs
 
   def find_plans(coverage_kind)
     if coverage_kind == 'dental'
-      plans = @benefit_group.elected_dental_plans_by_option_kind.to_a
+      plans = [@benefit_group.dental_reference_plan]
       plans.sort_by!(&:name) && ([@benefit_group.dental_reference_plan] + plans).uniq
```

Comment:
as we're only showing one product; you can remove this.

Reply:
changes add.